apps/indexer/lib/indexer/fetcher/internal_transaction.ex

```diff
@@ -121,43 +121,74 @@ defmodule Indexer.Fetcher.InternalTransaction do
           error_count: unique_numbers_count
         )
 
-        errored =
-          reason
-          |> Enum.map(fn
-            %{data: %{block_number: num}} -> num
-            %{data: %{"blockNumber" => num}} -> num
-          end)
+        # errored =
+        #  reason
+        #  |> Enum.map(fn
+        #    %{data: %{block_number: num}} -> num
+        #    %{data: %{"blockNumber" => num}} -> num
+        #  end)
 
-        result = Chain.bump_pending_blocks(errored)
+        # result = Chain.bump_pending_blocks(errored)
```

Comment:
Why not bump them?

Reply:
I think we should remove bumping, the archive nodes seem to work fine now so it's not needed anymore.

Reply:
Let's remove the commented out code then?